react.md

```diff
@@ -0,0 +1,137 @@
+# Introduction
+
+[React](https://facebook.github.io/react/) is a JavaScript library for building user interfaces. The following guide is to be followed in addition to the [ES2015](es2015.md) style guide.
+
+# Tooling
+
+To lint our React apps we use:
+
+- [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react)
+
+To plug into [eslint](http://eslint.org).
+
+# Styleguide
+
+General points:
+
+- React Props and States are camelCased
+- Use propTypes where practicable
+- Single line JSX to have parentheses
+- Multiline props when more than 5 props are in a component
```

Comment:
Personally, I try to fit props into the 100 characters per line and if it turns out that 1 has to go on a new line or it looks/feels weird I will put each of them on a new line.